### DISCUSSION TITLE
Links not from center to center

### DISCUSSION THREAD

**Discussion by Roemeo** [matteobruni/tsparticles#5143] [Q&A] [answered]
Hello,

I hope I have not overlooked anything. My particle are circles with a transparent fill. My problem is, that the links are drawn from center to center, so one can see these in the transparent inner of the circle. One should see the background through them. I want the links only to touch the border of my particle. Is it somehome possibly to modify the links? 

Kind regards

Fabian Römer

**Comment by matteobruni** [comment, marked as answer]
You need to create a custom plugin for that behavior. You can start from the [existing one](https://github.com/matteobruni/tsparticles/tree/main/interactions/particles/links) and customize it yourself.
